src/app/Base.tsx

```
@@ -88,9 +89,9 @@ export const Base: React.FC<BaseProps> = ({ children }) => {
 							paddingTop:
 								width > 1000
 									? "76px"
-									: path === "/"
+									: path == "/"
```

Biome (1.9.4)

[error] Use === instead of ==. == is only allowed when comparing against `null`

== is only allowed when comparing against null

Using == may be unsafe if you are relying on type coercion
Unsafe fix: Use ===


(lint/suspicious/noDoubleEquals)